File: aiJ-client/assets/Script/plazz/event/RoomRecordEvent.ts
import {AiJ} from "../../ws/AiJ";

/**
 * 房间游戏记录
 */
export default class RoomRecordEvent extends AiJ.AiJEvent {
    /**
     * 页
     */
    page: number;
    /**
     * 页大小
     */
    pageSize: number;

    constructor(page: number, pageSize: number) {
        super();
        this.page = page;
        this.pageSize = pageSize;
        this.mainType = 3;
        this.subType = 2;
    }
}

File: aiJ-plaza/src/main/java/com/xiyoufang/aij/plaza/event/RoomRecordEvent.java
package com.xiyoufang.aij.plaza.event;

import com.xiyoufang.aij.event.Event;

/**
 * Created by 席有芳 on 2019-01-30.
 * 获取游戏记录
 *
 * @author 席有芳
 */
public class RoomRecordEvent extends Event {
    /**
     * 页码
     */
    private int page;
    /**
     * 每页显示条数
     */
    private int pageSize;

    public int getPage() {
        return page;
    }

    public void setPage(int page) {
        this.page = page;
    }

    public int getPageSize() {
        return pageSize;
    }

    public void setPageSize(int pageSize) {
        this.pageSize = pageSize;
    }
}
